src/helpers/comment.ts

```diff
@@ -0,0 +1,73 @@
+import { fromMarkdown } from "mdast-util-from-markdown";
+import { gfmFromMarkdown } from "mdast-util-gfm";
+import { gfm } from "micromark-extension-gfm";
+
+export const MarkdownItem = {
+  Paragraph: "paragraph",
+  List: "list",
+  Link: "link",
+  Text: "text",
+  Code: "code",
+} as const;
+
+const PriceItem: Record<string, number> = {
+  [MarkdownItem.Text]: 0.1,
+  [MarkdownItem.Link]: 0.5,
```

Comment:
Is there a difference between bullet and numbered lists in `mdast`? Or all of them are parsed as a `List` type? I mean is there smth like `List.Bullet` or `List.Numbered` ?

Reply:
I think it is okay to consolidate both list types then.

Reply:
I couldn't find both `List.Bullet` and `List.Numbered` so I guess that's fine to consolidate both list types as of now.